spec/requests/content_item_requests/derived_representations_spec.rb

```diff
@@ -1,6 +1,10 @@
 require "rails_helper"
 
 RSpec.describe "Derived representations", type: :request do
+  before do
+    login_as_stub_user
```

Comment:
This is needed on all request specs, so it can be configured in the spec helper:
```
  config.before :each, type: :request do
    login_as_stub_user
  end
```